src/fontra/views/fontoverview/fontoverview.js

```diff
@@ -340,6 +340,19 @@ export class FontOverviewController extends ViewController {
     const glyphItemList = this.glyphOrganizer.filterGlyphs(combinedGlyphItemList);
     const glyphSections = this.glyphOrganizer.groupGlyphs(glyphItemList);
     this.glyphCellView.setGlyphSections(glyphSections);
+
+    // Add placeholder if no glyphs are found
+    if (glyphSections.length === 0) {
+      if (!this.glyphCellViewContainer.querySelector(".font-overview-no-glyphs")) {
```

Comment:
How about we add this element at the same time as the glyphCellView, and just leave it there with empty content?
The we don't need `glyphCellViewContainer` as a property, and we'd add `this.noGlyphsElement` with fixed content, and set `display: none` in the CSS, and add a class if there are indeed no glyphs, which will make the display set to normal.

Reply:
Please see the latest commit.